client/app/css/source.less

```diff
@@ -107,3 +107,7 @@
   max-width: inherit;
   margin-bottom: 0;
 }
+
+h3.font-lite {
```

Comment:
Please use https://getbootstrap.com/docs/4.1/utilities/colors/#color instead

Reply:
The reason why this class was added was to reduce the font-weight so that the text fits the available space and does not wrap the exclamation to the next line.
Bootstrap has a class 'font-weight-light' that has a lower font weight and is sufficient. But we are not able to apply this class to the element because the element has it's font weight already being set by the tag name as follows:
h3 {
    font-weight: 400;
}

Since there is the above css in the wizard step component, we are not able to override the css using a css class. Hence the only way to get it working is to create another css by tagname.

Reply:
Setting `white-space: nowrap` would cause the text to overflow the parent container. But since the text here is limited, we can go ahead with it.

Reply:
Thanks @ganeshkumar1989.

I will take another look with a `Non-breaking space` directly on the XML file but until now, let's move on with this one.

Reply:
Thanks @antleblanc :)